From: "Rafael J. Wysocki" <rafael.j.wysocki@intel.com>
Date: Thu, 25 Apr 2024 16:07:38 +0200
Subject: thermal: trip: Add missing empty code line
Patch-mainline: v6.10-rc1
Git-commit: 1502718a66c573cb2df811da67d8de7bf5c4657e
References: jsc#PED-11164

Add missing empty line of code to thermal_zone_trip_id().

Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Reviewed-by: Lukasz Luba <lukasz.luba@arm.com>
Acked-by: Chun-Yi Lee <jlee@suse.com>
---
 drivers/thermal/thermal_trip.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/thermal/thermal_trip.c
+++ b/drivers/thermal/thermal_trip.c
@@ -138,6 +138,7 @@ int thermal_zone_trip_id(const struct th
 	 */
 	return trip_to_trip_desc(trip) - tz->trips;
 }
+
 void thermal_zone_trip_updated(struct thermal_zone_device *tz,
 			       const struct thermal_trip *trip)
 {
